core/extension/gdextension.cpp

```diff
@@ -41,6 +41,11 @@ extern GDExtensionInterfaceFunctionPtr gdextension_get_proc_address(const char *
 
 typedef GDExtensionBool (*GDExtensionLegacyInitializationFunction)(void *p_interface, GDExtensionClassLibraryPtr p_library, GDExtensionInitialization *r_initialization);
 
+#if defined(WINDOWS_ENABLED) && defined(TOOLS_ENABLED)
+// Prefix for temporary copies of libraries.
+#define TEMP_COPY_PREFIX "~"
```

Comment:
Not sure why it needs a define, it's used only once.

Reply:
True. I was using it more times at first but with the current state it's not really needed.